### ISSUE TITLE
'Clear image cache for selected movie(s)' removes files when caching is disabled

### ISSUE THREAD

**Issue by LeLobster** [tinyMediaManager/tinyMediaManager#153]
__What TMM version are you using?__
2.9.3.1 release

__What is the actual behaviour?__
When selecting `Clear image cache for selected movie(s)` while not having image cache enabled in the general settings will remove the actual .jpg files from the movie directory with no way to retrieve them.

__What is the expected behaviour?__
I expected TMM to refresh the movies' images to what is found locally on the disk

__Steps to reproduce:__
- Have `Enable image cache` unchecked in the general TMM options
- Select some already scraped movies and select `Clear image cache for selected movie(s)`

---
I have the cache disabled because my library is quite large and the images were taking up over a couple 100MB's in total.    
When I was adding a couple new movies to my library just now I locally edited some posters/fanart .jpg files and when the edits weren't showing in TMM I selected the clean option.  
This deleted the files resulting in me having to rescrape the movies for posters/fanart.  

Also of note: Selecting `Download missing artwork for selected movie(s)` doesn't do anything after the above happened because, I guess, the green checkmark for artwork is still present in TMM.

```
2017-05-22 00:41:12,168 INFO  [tmmpool-download-task-T1-G82] o.t.c.m.t.MovieMissingArtworkDownloadTask:60 - Getting missing artwork
2017-05-22 00:41:12,170 INFO  [tmmpool-download-task-T1-G82] o.t.c.m.t.MovieMissingArtworkDownloadTask:71 - Done getting missing artwork
```

**Comment by myron0815** [comment]
fixed the image cache thingy for next release;
need to check for artwork download...

**Comment by LeLobster** [comment]
I don't think that's really needed, if the main issue is fixed their shouldn't be any other cases where the artwork is missing while TMM still thinks it exists. Unless the user manually deletes the jpg files.

**Comment by mlaggner** [comment]
this should be fixed with 2.9.4
